playwright.config.ts

```diff
@@ -21,14 +21,13 @@ const config: PlaywrightTestConfig = {
      */
     timeout: 5000,
   },
-  /* Run tests in files in parallel */
-  fullyParallel: true,
+  /* Disable parallelism to avoid the `nonce has already been used` issue */
```

Comment:
I fixed that in different way: https://github.com/cawabunga/headless-web3-provider/pull/13